src/variants.php

```diff
@@ -69,36 +69,46 @@
 
 
 
-if (!ACTION && (empty($_PE[1]) || preg_match('/^chr[0-9A-Z]{1,2}$/', $_PE[1]))) {
+if (!ACTION && (empty($_PE[1]) ||
+    preg_match('/^(chr[0-9A-Z]{1,2})(:([0-9,]+)-([0-9,]+))?$/', $_PE[1], $aRegionArgs))) {
```

Comment:
I agree. This is not the place to allow commas in the region description. If needed, that can be added in the front-end later on. Fixed in 7881a95